server/dist/index.js

```diff
@@ -17,6 +17,13 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 var app = (0, _express2.default)();
 var port = 3030;
 
+app.use(function (req, res, next) {
```

Comment:
You probably don't want to subject files in the `dist` directory to version control, as `dist` should contain a bundled package of your client application.